packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesUtils.kt

```diff
@@ -185,7 +185,7 @@ object UpdatesUtils {
   fun bytesToHex(bytes: ByteArray): String {
     val hexChars = CharArray(bytes.size * 2)
     for (j in bytes.indices) {
-      val v = (bytes[j] and 0xFF.toByte()).toInt()
+      val v = bytes[j].toInt() and 0xFF
```

Comment:
can we use kotlin `ByteArray.toHexString` than implement this by ourselves?

Reply:
I guess the main concern for using `ByteArray.toHexString` here is that it was only added in Kotlin 1.9 and it's still a [@ExperimentalStdlibApi](https://kotlinlang.org/api/core/1.9/kotlin-stdlib/kotlin/-experimental-stdlib-api/index.html), but if we're ok with this we could migrate over

Reply:
fair enough. it looks like kotlin toHexString has more complex supported cases in mind.